s3.go

```diff
@@ -99,7 +100,16 @@ func (m *S3RegionManagedDownloader) Download(dr *DownloadRecord, localFile *os.F
 		return fmt.Errorf("Unable to get downloader for %s: %s", bucket, err)
 	}
 
-	log.Debugf("Downloading s3://%s/%s", bucket, fname)
+	var requestID, hostID string
+	requestInspectorFunc := func(r *request.Request) {
+		r.Handlers.Complete.PushBack(func(req *request.Request) {
+			requestID = req.RequestID
+			if req.HTTPResponse != nil && req.HTTPResponse.Header != nil {
+				hostID = req.HTTPResponse.Header.Get("X-Amz-Id-2")
```

Comment:
Great! I was knowing about the HTTP Header `x-amz-request-id` but not about [x-amz-id-2](https://docs.aws.amazon.com/AmazonS3/latest/API/RESTCommonResponseHeaders.html) 👍

Reply:
I just looked in their lib for `HostID` :)